tests/e2e/configs/one-chain.yaml

```diff
@@ -2,6 +2,7 @@ chains:
   - name: osmosis-1
     type: osmosis
     numValidators: 1
+    metrics: true
```

Comment:
we should enabled metrics by default i think.

Reply:
In my mind Starship users running it locally don't care about metrics. If you run it in production, then it's required. 

I don't know how much additional overhead it adds if you enable it by default. If it's low, then it shouldn't be a problem to enable it for everyone.

Reply:
Makes sense. Lets keep it as a flag then.